gapic/ads-templates/setup.py.j2

```diff
@@ -29,7 +29,8 @@ else:
     release_status = "Development Status :: 5 - Production/Stable"
 
 dependencies = [
-    "google-api-core[grpc] >= 2.10.0, < 3.0.0dev",
+    "google-api-core @ git+https://github.com/googleapis/python-api-core.git@add-api-version-header",
+    #"google-api-core[grpc] >= 2.10.0, < 3.0.0dev",
```

Comment:
Revert once https://github.com/googleapis/python-api-core/pull/638 is merged and released to PyPI

Reply:
Obsolete